### ISSUE TITLE
Missing props for accessibility

### ISSUE THREAD

**Issue by YuridiaLarios** [ankeetmaini/react-infinite-scroll-component#411]
I am trying to add role="list" to the react-infinite-scroll-component top div so screenreaders can announce how many children are under this list, however there is no way to add it under the existing props so screenreaders cannot announce items inside the infinity scroll list correctly.
